includes/acl/algorithm/uniformly_sampled/decoder.h

```diff
@@ -61,13 +61,14 @@ namespace acl
 
 		namespace impl
 		{
-			// TODO: Ensure RO data is aligned to 64 bytes and RW data is aligned to 64 bytes as well
-			struct DecompressionContext
+			static constexpr size_t CONTEXT_ALIGN_AS = 64;
+
+			struct alignas(CONTEXT_ALIGN_AS) DecompressionContext
 			{
 				// Read-only data
-				const uint32_t* default_tracks_bitset;
```

Comment:
Yeah this is pretty much what I had in mind. A few things missing but good enough to merge.
Things missing: a clip pointer here to validate that when we attempt to use the context, it belongs to the current clip. It is common to switch clips at runtime within an animation node and forgetting to refresh the context might not be unusual. I would also introduce an interface for the decompression context, IDecompressionContext for debugging purposes. We only need a virtual destructor that does nothing in order for us to have a vtable. With a vtable, the debugger will know the actual object type and will allow you to see the members inside without setting deep in here where we actually use it. This also ensures minimal type safety since anything could safely coerce to a void*. An actual type should be introduced so we might as well make it virtual. The actual cost of the vtable isn't that big a deal. If we really were worried, we could align the first member of the derived type. the vtable will actually never be touched at runtime (except when we delete the context) so it won't mess with cache alignment